flink-connectors/flink-sql-connector-elasticsearch7/src/main/resources/META-INF/NOTICE

```diff
@@ -15,31 +15,33 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.httpcomponents:httpasyncclient:4.1.4
 - org.apache.httpcomponents:httpclient:4.5.3
 - org.apache.httpcomponents:httpcore:4.4.6
-- org.apache.httpcomponents:httpcore-nio:4.4.11
+- org.apache.httpcomponents:httpcore-nio:4.4.12
 - org.apache.logging.log4j:log4j-api:2.11.1
 - org.apache.logging.log4j:log4j-to-slf4j:2.11.1
-- org.apache.lucene:lucene-analyzers-common:8.1.0
-- org.apache.lucene:lucene-backward-codecs:8.1.0
-- org.apache.lucene:lucene-core:8.1.0
-- org.apache.lucene:lucene-grouping:8.1.0
-- org.apache.lucene:lucene-highlighter:8.1.0
-- org.apache.lucene:lucene-join:8.1.0
-- org.apache.lucene:lucene-memory:8.1.0
-- org.apache.lucene:lucene-misc:8.1.0
-- org.apache.lucene:lucene-queries:8.1.0
-- org.apache.lucene:lucene-queryparser:8.1.0
-- org.apache.lucene:lucene-sandbox:8.1.0
-- org.apache.lucene:lucene-spatial:8.1.0
-- org.apache.lucene:lucene-spatial-extras:8.1.0
-- org.apache.lucene:lucene-spatial3d:8.1.0
-- org.apache.lucene:lucene-suggest:8.1.0
-- org.elasticsearch:elasticsearch:7.3.2
-- org.elasticsearch:elasticsearch-cli:7.3.2
-- org.elasticsearch:elasticsearch-core:7.3.2
-- org.elasticsearch:elasticsearch-secure-sm:7.3.2
-- org.elasticsearch:elasticsearch-x-content:7.3.2
-- org.elasticsearch.client:elasticsearch-rest-client:7.3.2
-- org.elasticsearch.client:elasticsearch-rest-high-level-client:7.3.2
-- org.elasticsearch.plugin:aggs-matrix-stats-client:7.3.2
-- org.elasticsearch.plugin:parent-join-client:7.3.2
-- org.elasticsearch.plugin:rank-eval-client:7.3.2
+- org.apache.lucene:lucene-analyzers-common:8.3.0
+- org.apache.lucene:lucene-backward-codecs:8.3.0
+- org.apache.lucene:lucene-core:8.3.0
+- org.apache.lucene:lucene-grouping:8.3.0
+- org.apache.lucene:lucene-highlighter:8.3.0
+- org.apache.lucene:lucene-join:8.3.0
+- org.apache.lucene:lucene-memory:8.3.0
+- org.apache.lucene:lucene-misc:8.3.0
+- org.apache.lucene:lucene-queries:8.3.0
+- org.apache.lucene:lucene-queryparser:8.3.0
+- org.apache.lucene:lucene-sandbox:8.3.0
+- org.apache.lucene:lucene-spatial:8.3.0
+- org.apache.lucene:lucene-spatial-extras:8.3.0
+- org.apache.lucene:lucene-spatial3d:8.3.0
+- org.apache.lucene:lucene-suggest:8.3.0
+- org.elasticsearch:elasticsearch:7.5.1
+- org.elasticsearch:elasticsearch-cli:7.5.1
+- org.elasticsearch:elasticsearch-core:7.5.1
+- org.elasticsearch:elasticsearch-secure-sm:7.5.1
+- org.elasticsearch:elasticsearch-x-content:7.5.1
+- org.elasticsearch.client:elasticsearch-rest-client:7.5.1
+- org.elasticsearch.client:elasticsearch-rest-high-level-client:7.5.1
+- org.elasticsearch.plugin:aggs-matrix-stats-client:7.5.1
+- org.elasticsearch.plugin:lang-mustache-client:7.5.1
```

Comment:
Why did you add `mustache-client` here? I think the rest of the dependencies look correct but about this one I don't know. It doesn't appear in the list of jars that are shaded in when I run `mvn clean install -DskipTests -Dfast`. In the ES7 SQL connector module.

Reply:
I run `mvn dependency:tree` and it shows this plugin.

Reply:
But it's not in the resulting jar. If you run `mvn clean install -DskipTests -Dfast` in the ES7 SQL module you will get some output like this:

```
[INFO] Including org.apache.flink:flink-connector-elasticsearch7_2.11:jar:1.11-SNAPSHOT in the shaded jar.
[INFO] Including org.apache.flink:flink-connector-elasticsearch-base_2.11:jar:1.11-SNAPSHOT in the shaded jar.
[INFO] Including org.elasticsearch.client:elasticsearch-rest-high-level-client:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch:elasticsearch:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch:elasticsearch-core:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch:elasticsearch-secure-sm:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch:elasticsearch-x-content:jar:7.5.1 in the shaded jar.
[INFO] Excluding org.yaml:snakeyaml:jar:1.17 from the shaded jar.
[INFO] Including com.fasterxml.jackson.core:jackson-core:jar:2.10.1 in the shaded jar.
[INFO] Including com.fasterxml.jackson.dataformat:jackson-dataformat-smile:jar:2.8.11 in the shaded jar.
[INFO] Including com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.8.11 in the shaded jar.
[INFO] Including com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar:2.8.11 in the shaded jar.
[INFO] Excluding org.elasticsearch:elasticsearch-geo:jar:7.5.1 from the shaded jar.
[INFO] Including org.apache.lucene:lucene-core:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-analyzers-common:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-backward-codecs:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-grouping:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-highlighter:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-join:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-memory:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-misc:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-queries:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-queryparser:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-sandbox:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-spatial:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-spatial-extras:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-spatial3d:jar:8.3.0 in the shaded jar.
[INFO] Including org.apache.lucene:lucene-suggest:jar:8.3.0 in the shaded jar.
[INFO] Including org.elasticsearch:elasticsearch-cli:jar:7.5.1 in the shaded jar.
[INFO] Excluding net.sf.jopt-simple:jopt-simple:jar:5.0.2 from the shaded jar.
[INFO] Excluding com.carrotsearch:hppc:jar:0.8.1 from the shaded jar.
[INFO] Excluding joda-time:joda-time:jar:2.5 from the shaded jar.
[INFO] Excluding com.tdunning:t-digest:jar:3.2 from the shaded jar.
[INFO] Excluding org.hdrhistogram:HdrHistogram:jar:2.1.9 from the shaded jar.
[INFO] Excluding org.elasticsearch:jna:jar:4.5.1 from the shaded jar.
[INFO] Including org.elasticsearch.client:elasticsearch-rest-client:jar:7.5.1 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpclient:jar:4.5.3 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore:jar:4.4.6 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpasyncclient:jar:4.1.4 in the shaded jar.
[INFO] Including org.apache.httpcomponents:httpcore-nio:jar:4.4.12 in the shaded jar.
[INFO] Including commons-codec:commons-codec:jar:1.10 in the shaded jar.
[INFO] Including commons-logging:commons-logging:jar:1.1.3 in the shaded jar.
[INFO] Including org.elasticsearch.plugin:mapper-extras-client:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch.plugin:parent-join-client:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch.plugin:aggs-matrix-stats-client:jar:7.5.1 in the shaded jar.
[INFO] Including org.elasticsearch.plugin:rank-eval-client:jar:7.5.1 in the shaded jar.
[INFO] Excluding org.elasticsearch.plugin:lang-mustache-client:jar:7.5.1 from the shaded jar.
[INFO] Excluding com.github.spullara.mustache.java:compiler:jar:0.9.6 from the shaded jar.
[INFO] Including org.apache.logging.log4j:log4j-to-slf4j:jar:2.11.1 in the shaded jar.
[INFO] Including org.apache.logging.log4j:log4j-api:jar:2.11.1 in the shaded jar.
[INFO] Including org.apache.flink:force-shading:jar:1.11-SNAPSHOT in the shaded jar.
[INFO] No artifact matching filter io.netty:netty
```
which tells you what is included in the jar.

Reply:
OK, Got it. I referenced the dependency tree before. Will remove it now.

```
- org.elasticsearch.plugin:lang-mustache-client:7.5.1
```

Reply:
Done.

Reply:
Thanks, I think this is good to merge now! 🍵